src/Umbraco.Core/Persistence/Migrations/Upgrades/TargetVersionSevenFourZero/AddDataDecimalColumn.cs

```diff
@@ -0,0 +1,29 @@
+using System.Linq;
+using Umbraco.Core.Configuration;
+using Umbraco.Core.Logging;
+using Umbraco.Core.Persistence.SqlSyntax;
+
+namespace Umbraco.Core.Persistence.Migrations.Upgrades.TargetVersionSevenFourZero
+{
+    [Migration("7.4.0", 1, GlobalSettings.UmbracoMigrationName)]
+    public class AddDataDecimalColumn : MigrationBase
+    {
+        public AddDataDecimalColumn(ISqlSyntaxProvider sqlSyntax, ILogger logger)
+            : base(sqlSyntax, logger)
+        {
+        }
+
+        public override void Up()
+        {
+            //Don't exeucte if the column is already there
+            var columns = SqlSyntax.GetColumnsInSchema(Context.Database).ToArray();
+
+            if (columns.Any(x => x.TableName.InvariantEquals("cmsPropertyData") && x.ColumnName.InvariantEquals("dataDecimal")) == false)
+                Create.Column("dataDecimal").OnTable("cmsPropertyData").AsDecimal(18, 9).Nullable();
```

Comment:
It seems this fails on SQL server with the error:

ALTER TABLE [cmsPropertyData] ADD [dataDecimal] DECIMAL(38,6)(18) NULL

Incorrect syntax near '18'.

... I'll fix it up, just reporting